force-app/main/selector/DonationHistorySelector.cls

```diff
@@ -131,6 +131,13 @@ public inherited sharing class DonationHistorySelector {
         return totalRecords;
     }
 
+    /** 
+    * @description method to return Amount or converCurrency(Amount) field if org is multicurrency. 
+    */
+    public String isMultiCurrencyOrganization(){
```

Comment:
This method name sounds like it would return a Boolean, can we use `getOppAmountFieldForQuery` or something like that? Also, could it include the `String.escapeSingleQuotes` portion so we don't need to include that in the query itself? Thanks!